### PR TITLE
spring-vasylpekhterev

### PR DESCRIPTION
**Pull request by vasiliypekhterev**

### CODE REVIEW

File: VasylPekhterev/src/main/java/spring/aspect/LogginAspect.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+        Object value = null;
+        try {
+            long startTime = System.currentTimeMillis();
+            value = proceedingJoinPoint.proceed();
+            long endTime = System.currentTimeMillis();
+            LOGGER.info(String.format("service method %s.%s was run %s milliseconds\n",
+                    typeName, methodName, endTime-startTime));
+        } catch (Throwable throwable) {
+            throwable.printStackTrace();
```
Comment (presly808) [inline VasylPekhterev/src/main/java/spring/aspect/LogginAspect.java]: you should rethrow exception, without it you intercept an exception

File: VasylPekhterev/src/main/java/spring/dao/UserDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import javax.persistence.PersistenceContext;
+
+@Component
+public class UserDaoImpl implements IUserDao {
+
+    @PersistenceContext
+    private EntityManager manager;
+
+    @Override
+    @Transactional
```
Comment (presly808) [inline VasylPekhterev/src/main/java/spring/dao/UserDaoImpl.java]: Transactional should be on a business logic layer(Service class)

File: VasylPekhterev/src/test/java/spring/service/IUserServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import spring.model.User;
+
+import javax.persistence.EntityManagerFactory;
+
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.junit.Assert.*;
+
+public class IUserServiceTest {
```
Comment (presly808) [inline VasylPekhterev/src/test/java/spring/service/IUserServiceTest.java]: use spring-test module for testing

Comment (vasiliypekhterev) [reply VasylPekhterev/src/test/java/spring/service/IUserServiceTest.java]: all remarks was fixed

**Pull request merged**